arwen/context/arwen.go

```diff
@@ -187,13 +195,13 @@ func (host *vmContext) RunSmartContractCall(input *vmcommon.ContractCallInput) (
 
 	var err error
 	opcode_costs := host.GasSchedule().WASMOpcodeCost.ToOpcodeCostsArray()
-	host.instance, err = wasmer.NewMeteredInstanceWithImports(contract, host.imports, uint64(gasLeft), &opcode_costs)
+	host.instance, err = wasmer.NewMeteredInstanceWithImportObject(contract, &host.importObject, uint64(gasLeft), &opcode_costs)
```

Comment:
make op_code cost to be instantiated only once at arwen creation.

Reply:
Done.